packages/client/src/components/TextLink.js

```diff
@@ -11,9 +12,7 @@ const TextLink = (props) => {
       padding={props.padding}
       onClick={props.onClick}
     >
-      <Link to={props.link}>
-        <span>{props.text}</span>
-      </Link>
+      <span onClick={() => history.push(props.onClickGoTo)}>{props.text}</span>
```

Comment:
I actually think using Link works better here.

Reply:
cool, I'll revert this and do another commit in a bit!